### DISCUSSION TITLE
PPS integration

### DISCUSSION THREAD

**Discussion by Caribicjoe** [oxplot/fpx#23] [Ideas]
Hi,

would it be possible to integrate PPS functionality and therefore utalize even 10mV steps? If theres already a project as easy to use, small and versitile, as yours and i´ve not seen it, im sorry :-) (compliments intended ;-))

**Comment by oxplot** [comment]
Possible, yes, but I'm already working on a new version of the board with different software, which does in fact support PPS. As for fpx, I don't plan to make any further changes.

**Reply by Caribicjoe** [reply]
hi, thanks for the reply. how can I find it once you´re finished? will it also be open source?
might be able to help, i´m an electrical engineer

**Reply by oxplot** [reply]
Yes, it'll be open source.

I'll make a link to it from fpx repo as well as my Tindie store. But don't hold your breath, it'll be at least a few months before that happens (and most likely about 6 months).

**Reply by oxplot** [reply]
Appreciate the help. But I should be ok :)
